lib/fakerbot/renderer.rb

```diff
@@ -73,5 +64,20 @@ def gt_screen_height?(data_tree)
     def verbose?
       options[:verbose]
     end
+
+    def verbose_output(method, const, arr)
+      fake, message = faker_method(method, const)
+      arr << crayon.dim.white("=> #{fake.to_s}") << crayon.dim.magenta.bold("#{message}")
+    end
+
+    def faker_method(method, const)
+      [const.public_send(method), deprecate(method, const)]
+    rescue ArgumentError
+      ['N/A', '']
+    end
+
+    def deprecate(method, const)
```

Comment:
What are your thoughts on renaming this method to `ensure_method_is_supported` or sth similar for clarity?

Reply:
I couldn't pick up `ensure_..`  from my brain. 😮 
I remembered!